lib/spack/spack/test/install.py

```diff
@@ -238,13 +238,11 @@ def test_install_dependency_symlinks_pkg(install_mockery, mock_fetch, mutable_mo
 
 def test_install_times(install_mockery, mock_fetch, mutable_mock_repo):
```

Comment:
I don't think this test makes much sense without the phases defined in `dev-build-test-install-phases` (and if it's passing, I have no idea how).

Reply:
I think this tests the format of the `timefile`? As such it  still makes sense. I'll have a look if this can be reverted without issues, since the transformation here was done before we decided to allow custom phases.